README.md

```diff
@@ -33,65 +32,198 @@ Only time will tell if making our product roadmap
+containing several features
+that together help the individual
+achieve _all_ of their personal goals
+through a systematic workflow.
+
+The @dwyl App Will have the following basic workflow:
+***Capture, Categorise Complete***.
+
+1. ***Capture*** - capture everything that is on your mind in a "brain dump".
+2. ***Categorise*** - split out blocks of text
+  into distinct items and _categorise_ them. <br />
+  **a)** What type of info/item is it?
+  e.g: Todo Item, Reading list item, shopping list,
+  random thought, bucket list? <br />
+  **b)** Do you have an _existing_ category/tag for the item
+  or is it **`new`**? <br />
+  **c)** Contextualise the item in terms of time/space:
+  a shopping list item context could be "grocery store"
+  whereas a reading list item could be "daily commute"<br />
+  **d)** **Add** any additional ***detail*** you can think of at the time.
+  e.g: acceptance criteria, deadline, dependencies <br />
+  **e)** If the item is a task that has sub-tasks, create and link them.
+3. ***Complete*** - do the work to _complete_ the item/task.<br />
+  **a)** Re-read the original description
+  and determine if you have everything you need to start the task.
+  If not, write down what is missing. <br />
+  **b)** ***Delegate*** if necessary/possible <br />
+  **c)** ***Track progress*** of the task. <br />
+  **d)** ***Confirm*** it was completed to expected standard. <br />
+  **e)** ***Review*** & ***Reflect*** - confirm the task was completed
+  and all the original criteria were met by the assignee.
+  Leave a brief feedback comment with any thoughts.
+  Check the item off as "Done" if appropriate.
+
+For the first step (***Capture***),
+we will build an ultra-simple interface
+to allow people to
+input **`palintext`**
+and save it to a backend as **`text`**.
+For details, please see:
+[`brain-dump.md`](https://github.com/dwyl/product-roadmap/blob/master/checklists.md)
+
+Once the text input is working,
+we will iterate and improve it
+in several interesting and innovative ways
+in response to feedback from people _using_ the MVP. <br />
+We will add additional methods
+to help people "capture" information.
+e.g:
++ Linking to external sources of information.
++ Photo/image uploads
+  + Optical Character Recognition (OCR)
+  + Image/object recognition
++ Voice/audio recording
+  + Speech transcription and indexing
+
+These are complex features
+that have formed the basis for entire companies.
+So in order to make progress with them,
+we will need to _significantly_ leverage
+the Open Source work of many brilliant people.
+But we are getting ahead of ourselves;
+we need to _first_ focus
+on creating "MVP" versions
+of each step in the workflow.
+
+For the _second_ step, ***Categorise***,
+we will need a tagging system that allows people
+to apply textual labels to items.
+The UX for this is still "TBD".
+We will need to sketch it out ahead of sprint planning.
+See: https://github.com/dwyl/product-roadmap/issues/15
```

Comment:
```suggestion
See: https://github.com/dwyl/app/issues/235
```

This issue has now been moved to the dwyl/app repo (github is doing the redirection automatically but it might still be worth to have the correct link)